samples/Mvc.Client/project.json

```diff
@@ -34,7 +34,8 @@
         "Microsoft.Framework.NotNullAttribute.Internal": {
             "type": "build",
             "version": "1.0.0-*"
-        }
+        },
+        "AspNet.Security.OAuth.FourSquare": "1.0.0-*"
```

Comment:
Don't forget to rename `FourSquare` to  `Foursquare` everywhere (namespaces and file names)